src/components/media/MuiAvatarShim.jsx

```diff
@@ -13,7 +13,7 @@ const VALID_SIZES = ['s', 'm', 'l', 'xl'];
  */
 const MuiAvatarShim = ({ name, src, size, className, style }) => {
   const classNames = cx(
-    'fdsAvatar',
+    'fdsMUIAvatar',
```

Comment:
I would vote to remove this component with the introduction of `Avatar` - replace usage of `MuiAvatarShim` with `Avatar`

Reply:
totally. So my only question here: MuiAvatarShim supports large and extra large, which Avatar does not right now. I currently don't see any usage of `AvatarRow` in cbi-site. 

We could totally either:

-  deprecate the `large` and `xl` sizes that AvatarRow supports (this would require a version bump)
- Add support for `large` and `extra large` in Avatar... but the question there would be... do we need those.

Reply:
good call - removing `l` and `xl` would require a version bump, but deprecating them would not.

We could map unsupported values in `AvatarRow` to supported avatar sizes for now

```
// in `AvatarRow`
const sizeMap = {
  xs: 's',
  s: 's',
  m: 'm',
  l: 'm',
  xl: 'm',
};
```